### PR TITLE
fix: destroy vmss dependency

### PR DESCRIPTION
**Pull request by denise89p**

### CODE REVIEW

File: vmss.tf [hunk 1]
```diff
@@ -161,11 +161,12 @@ resource "azurerm_role_assignment" "vm_role_assignment" {
 resource "null_resource" "force-delete-vmss" {
   triggers = {
     vmss_name       = azurerm_linux_virtual_machine_scale_set.vmss.name
-    rg_name         = var.rg_name
+    rg_name         = data.azurerm_resource_group.rg.name
```
Comment (assafgi) [inline vmss.tf:164]: why did you need to change this?

Comment (assafgi) [reply vmss.tf:164]: what makes it better?

Comment (denise89p) [reply vmss.tf:164]: from data you sure u get get the right rg

Comment (assafgi) [reply vmss.tf:164]: Isn't it the same?
You use here the var:
```
data azurerm_resource_group "rg"{
  name = var.rg_name
}
```
Also it requires additional api call.
